src/sentry/api/endpoints/organization_sessions.py

```diff
@@ -87,8 +87,8 @@ def build_sessions_query(
     @contextmanager
     def handle_query_errors(self):
         try:
-            # TODO: this context manager should be decoupled from `OrganizationEventsEndpointBase`?
-            with super().handle_query_errors():
+            # TODO: this context manager should be merged into util handle_query_error?
```

Comment:
Can you add more context to the `?` here? Is the only difference handling more types of errors and parsing the message?

Reply:
hmmm
Yea only difference is that these are extending the helper to add more exceptions
I'll remove the `TODO:` here instead since it's really more a question of how to clean this up